### DISCUSSION TITLE
[Working Times] Managing expected hours per month

### DISCUSSION THREAD

**Discussion by joney94** [kimai/Paid-plugins#189] [Questions & Answers]
Our employees have a monthly amount of hours fixed in their work contracts.
Naturally, each month will have a differing amount of work days. Therefore, the expected amount of work time defined in "Contract settings" ("erwartete Stundenzahl") would have to be changed every month manually, to result in a stable amount of hours expected for every employee?

As an example:
September '24 has 21 Work days (including public holidays)
Oktober '24 has 23 work days (incl. public holidays)
Therefore, an employee with a contract of 80 hours per months would have to work 3,81 hours/day in September, and ~3,48 hours/day in Oktober.
Does that imply that I would have to change his expected working times in contract settings at the beginning of each month? Or is there another workaround (or feature in planning)?

Thanks a ton :)
Jonas

**Comment by kevinpapst** [comment]
Hi Jonas,

I have "fixed contract times" on the list of ideas. 

But I cannot share whether this will be based on weekly or monthly (or both) times .
And I cannot share when this will be available.

I should have more time in the next months, as summer holidays are finally coming to an end...

Best regards
Kevin

**Reply by joney94** [reply]
Hi Kevin, thanks for your reply.
Even though everyone does probably think his/her request is important, I'd like to suggest to somewhat prioritize that feature.
Reason for that: I feel that nowadays, most companies will probably have a fixed amount of working hours/month in their contracts - so supporting that would definitely be a big help for a majority of users :)

Best, and thanks for your help!
Jonas

**Reply by kevinpapst** [reply]
> Even though everyone does probably think his/her request is important

Correct  💯 percent agree. I know that feeling 😁

> I feel that nowadays, most companies will probably have a fixed amount of working hours/month in their contracts 

You have a feeling and I do talk to companies about their requirements regularly. And while your requirement is not super rare, it is still far away from being used more often than classical xx hours per day contracts. 
But there is a reason this feature request exists and I will get to it at some point.
